CHANGELOG.md

```diff
@@ -15,6 +15,57 @@ it according to semantic versioning. For example, if your PR adds a breaking cha
 should change the heading of the (upcoming) version to include a major version bump.
 
 -->
+# 6.0.0
+
+## @rjsf/antd
+
+- Updated the `README.md` to show the theme snapshot with the latest theme UI from the playground, making it a link to the theme
```

Comment:
I think it would suffice to just have one entry in Dev/Docs/Playground for these changes